benefits/static/css/styles.css

```diff
@@ -174,6 +173,12 @@ footer .footer-links a {
   background: #212121;
 }
 
+.links div {
```

Comment:
Try to avoid ever setting a css style on an entire html element (like `div`). You can add a class here like `links-div` (or something descriptive to describe what div is).